applications/MeshMovingApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -53,6 +55,23 @@ void AddCustomUtilitiesToPython(pybind11::module& m) {
         .def("ProjectVirtualValues3D", &ExplicitFixedMeshALEUtilities::ProjectVirtualValues<3>)
         .def("UndoMeshMovement", &ExplicitFixedMeshALEUtilities::UndoMeshMovement);
 
+    py::class_<LinearTransform, LinearTransform::Pointer>(m, "LinearTransform")
+        .def(py::init<const array_1d<double,3>&, const double, const array_1d<double,3>&, const array_1d<double,3>&>())
+        .def(py::init<const array_1d<double,3>&, const array_1d<double,3>&, const array_1d<double,3>&>())
+        .def(py::init<const Quaternion<double>&, const array_1d<double,3>&, const array_1d<double,3>&>())
+        .def("SetRotation", static_cast<void (LinearTransform::*)(const array_1d<double,3>&, const double, const array_1d<double,3>&)>(&LinearTransform::SetRotation))
+        .def("SetRotation", static_cast<void (LinearTransform::*)(const array_1d<double,3>&, const array_1d<double,3>&)>(&LinearTransform::SetRotation))
+        .def("SetRotation", static_cast<void (LinearTransform::*)(const Quaternion<double>&, const array_1d<double,3>&)>(&LinearTransform::SetRotation))
```

Comment:
you could use lambdas

Reply:
These are overloads that I thought should be part of the cpp interface as well. Can you give an example of what you mean?

Reply:
sure they should be there
I am just saying that theoretically it is possible to use lambdas instead of fct pointers (I read lambdas easier than fct pointers)
https://github.com/KratosMultiphysics/Kratos/blob/99d061b441c74e9acc56e750d5303fcbe0c572fa/kratos/python/add_data_communicator_to_python.cpp#L105-L108

see also: https://pybind11.readthedocs.io/en/stable/classes.html#overloaded-methods 
The optimal solution would be to use `overload_cast` but this is C++14 and we are stuck with 11 :(